website/client/components/auth/registerLoginReset.vue

```diff
@@ -472,6 +491,21 @@ export default {
 
       this.login();
     },
+    calculateTextWidth () {
+      let socialButtons = document.getElementById('social-buttons-row');
```

Comment:
instead of using getElementById, vue's way to do this is to add a ref="refName" attribute to the el you want to access, then you'll be able to access the element with this.$refs.refName

Reply:
Thanks for the tip! I'll implement this.